src/clj/rems/json.clj

```diff
@@ -34,12 +34,20 @@
    {:modules [(JodaModule.)]
     :decode-key-fn str->keyword-or-number}))
 
+(def mapper*
```

Comment:
how about a TODO comment about `mapper*` and `parse-string*`?

Reply:
Yes I made an issue about it https://github.com/CSCfi/rems/issues/1943